### ISSUE TITLE
Extend Grafana dashboard to display "Daily Traffic"

### ISSUE THREAD

**Issue by Buddinski88** [pdreker/fritz_exporter#70]
Hi pdreker,

thank you for sharing your solution to gather informations from my FritzBox 7590 👍 

In the past i've used another Grafana dashboard with a influx db in the backend. There i have also the Daily Traffic and total Download/Upload.

Is it possible to extend the dashboard to see this informations?

![image](https://user-images.githubusercontent.com/50332516/168283332-952fed13-fde2-4952-ad71-ae1e496f2c09.png)

Best regards,
Buddinski88

**Comment by pdreker** [comment]
Prometheus is a bit "finicky", when it comes to metrics broken down to "last day" or by day. This means, that while it may be possible, the neccessary query may be rather complex and slow when checking the dashboard. The current dashboard is just untended to be a demonstration of the capabilities not neccessarily a "production ready" solution.

I'll keep this open - If I find the time to carve out a solution it will get published, but in the meantime, maybe someone else comes up with a simple solution to the "Daily volume query problem.

**Comment by pdreker** [comment]
I have looked into the daily/weekly/monthly issue and I must say, that prometheus is just not made for those types of queries. I have found no usable way to include those metrics or extract them from prometheus without creating an additional piece of software externally and even then all solutions were... not nice...

A new version of the dashboard was just uploaded which at least includes "Data Transfer rx/tx" for the last 24h, 7 days, 30 days and 365 days. These will always be from "now" and are not aligned to the calendar dates, but this is better than nothing I think.

Total Up/Down has not been added, as it would be unclear, what the "start time" is which makes the value not very useful. If you really want the "Total", simply configure one of the new Transfer panels for a timeframe larger than your prometheus retention.
